android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -904,25 +899,20 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
         Log.d(TAG, "create request to " + captive_portal_url + " for url: " + UrlUtil.sanitizeUrl(envoyUrl, envoyService))
 
         val sanitizedOriginal = UrlUtil.sanitizeUrl(originalUrl, envoyService)
+        val cacheFolder = cacheMap.get(originalUrl)
 
-        if (cacheMap.keys.contains(originalUrl)) {
+        if (!cacheFolder.isNullOrEmpty()) {
```

Comment:
I'm not sure i see how this is related to this pr?

Reply:
i think the previous version of the helper had a non-nullable cacheFolder argument.  i think i can revert this.